spec/javascripts/karma_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'open3'
+
+# Run karma and parse the output for failures.
+# The spec will fail if any karma test fails.
+describe 'Karma' do
+  let(:runner)  { Open3.capture3('karma', 'start') }
+  let(:output)  { runner[0] + runner[1] }
+  let(:status)  { runner[2].exitstatus }
+
+  it 'expects all karma tests to pass' do
```

Comment:
👍, but I'm not certain how this is going to integrate with test coverage analysis in the future.  My hope would be to integrate a tool such as https://github.com/istanbuljs/nyc